### PR TITLE
Feature/application message

### PR DESCRIPTION
**Pull request by esion**
* [x] basic thread list
* [x] tests (func/unit?)
* [x] responsive
* [x] squash commits and ready

caliopen_discussions

caliopen_discussion_responsive_sm

caliopen_discussions_responsive_xs

### CODE REVIEW

File: app/pods/messages/index/route.js [hunk 1]
```diff
@@ -2,6 +2,6 @@ import Ember from 'ember';
 
 export default Ember.Route.extend({
     model: function () {
-        return this.store.find('message');
+        return this.store.find('thread');
```
Comment (esion) [inline app/pods/messages/index/route.js]: @themouette  what about moving application "messages" to "thread" to fit with the API ? 

Then we'll add an new messages app which belongsTo a thread

Comment (esion) [inline app/pods/messages/index/route.js]: finally I'll a new app thread and make it default. selecting thread will render messages index route

Comment (themouette) [inline app/pods/messages/index/route.js]: What makes sense to me is to have the **message** application displaying all the message related views (ie threads and messages)

The routes tree should map the following:

* messages (app route)
  * index (display threads)
  * thread (route /:id)
    * index (display a single thread)
    * message (route /:id) display a single message
  * new (create a new conversation)

WDYT?

Comment (esion) [inline app/pods/messages/index/route.js]: I like it. 

may be "messages" becomes "discussions" to prevent misunderstood

Comment (themouette) [inline app/pods/messages/index/route.js]: nice idea!

**Comment by themouette** [comment]
Awesome, thank you

**Comment by esion** [comment]
\o/

**Pull request merged**